beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -445,7 +444,7 @@ func (vs *Server) validatorRegistered(ctx context.Context, id types.ValidatorInd
 }
 
 // Validates builder signature and returns an error if the signature is invalid.
-func (vs *Server) validateBuilderSignature(bid *ethpb.SignedBuilderBid) error {
+func (*Server) validateBuilderSignature(bid *ethpb.SignedBuilderBid) error {
```

Comment:
In that case we can change the signature to `func validateBuilderSignature(bid *ethpb.SignedBuilderBid) error`

Reply:
Done!